### ISSUE TITLE
One restcall to promote many deplyoments

### ISSUE THREAD

**Issue by andyzwa** [liimaorg/liima#353]
For promoting more than one deplyoment over the rest interface, it would be nicer to create all deployment in one rest call.
Actual for each deplyoment a (post)call have to be created.

**Comment by yvespp** [comment]
Related to #440

**Comment by mburri** [comment]
With the old frontend, it was possible to request multiple deployments at once:

![image](https://github.com/liimaorg/liima/assets/4577216/83352488-f468-4d0a-97f0-07ac6f231d04)

Note, that more than one deployment is checked in the first column.
When the button "redeploy" is clicked, a new screen allows to set parameters for the deployments:

![image](https://github.com/liimaorg/liima/assets/4577216/16e1ce44-9804-4e3f-a849-73cd90c12d84)

This allowed users to redeploy multiple applications and set the values (deployment date, environments, ...) only once.

With the new Frontend, redeployments are only possible for single deployments:

![image](https://github.com/liimaorg/liima/assets/4577216/f86973e4-7eab-4395-8d16-c31086131e6f)


If the user clicks "redeploy" the endpoint `POST /deployments` is called, implemented by `ch.mobi.itc.mobiliar.rest.deployments.DeploymentsRest#addDeployment`

We need to modify this endpoint (or add a new endpoint) that takes multiple applications and the configuration parameters to redeploy all applications at once. 
Then we need to change the front end to allow multiple deployments to be selected and redeployed.The selection is already there - but the redeploy action in the actions column only redeploys one application. I think we just need to add a "redeploy" button to the other buttons to initiate the workflow.

@yvespp are there other systems that use the current endpoint `POST /deployments` or is it only used by the angular GUI?

**Comment by yvespp** [comment]
@mburri The post deployments endpoint is used by the maven plugin (https://github.com/liimaorg/liima-maven-plugin) as well as the go cli (https://github.com/liimaorg/liimactl/blob/master/cmd/deployment/create.go)
We use both to create deployments from our CI/CD tool.
I also found some calls where no user agent is set, not sure who creates them...

**Comment by mburri** [comment]
@yvespp Thanks for your answer. In this case, we should probably add a new rest endpoint and not modify the existing one. 
I wonder if the new endpoint should allow both: single and multiple re-deployments.

**Comment by yvespp** [comment]
Talked to @andyzwa , it's no longer relevant for them. So i will close this issue for now.

In case we reopen it:
Discussed wir @mburri that it would make the most sens if you could just pass a list of deployment instead of just one. The go cli and the maven plugin already use appservername etc. for deployment and don't know about deployment ids.
